grammars/prql-lezer/src/prql.grammar

```diff
@@ -127,6 +137,14 @@ LambdaParam { identPart TypeDefinition? (":" expression)? }
   // underscores and allows no digit before the decimal point.
   space { " "+ }
 
+  Escape {
+    "\\" ("x" hex hex | "u" "{" hex+ "}" | ![xu])
```

Comment:
How would we handle `\n` at the moment?

Reply:
`\n` is given the `Escape` token and highlighted because `![xu]` matches any character that is not `x` or `u`. Unfortunately this includes `z` and others. This is the way the upstream JavaScript grammar does this though. Although on a second thought, I could replace that with `$[bfnrt]`.

Reply:
> `\n` is given the `Escape` token and highlighted because `![xu]` matches any character that is not `x` or `u`.

Ah yes. Great!